modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -430,6 +430,15 @@ const FDSEntries = ({
 		],
 	};
 
+	const onViewClick = ({itemData}: {itemData: FDSEntry}) => {
```

Comment:
Can't this just be `const onViewClick = ({itemData}: FDSEntry) => {` ?

Reply:
That would mean that `itemData` is a property of `FDSEntry`. TS Error:
```ts
Property 'itemData' does not exist on type 'FDSEntry'.
```

Reply:
Ah, okay, still rusty with TS